.github/workflows/deploy-staging.yaml

```
@@ -3,7 +3,7 @@ on:
   push:
     tags:
       # Matches stable and pre-releases
-      - "[0-9]+.[0-9]+.[0-9]+*"
+      - "v?[0-9]+.[0-9]+.[0-9]+*"
```

I suggest only expecting one format so we don't need to support two different tag formats:

```suggestion
      - "v[0-9]+.[0-9]+.[0-9]+*"
```